wazo_confd/plugins/line/api.yml

```diff
@@ -84,7 +84,11 @@ paths:
     delete:
       operationId: delete_line
       summary: Delete line
-      description: '**Required ACL:** `confd.lines.{line_id}.delete`'
+      description: '**Required ACL:** `confd.lines.{line_id}.delete`
+
+
+        **Disclaimer**: if `recursive=true`, the line is deleted, all their
```

Comment:
Out of curiosity is `recursive` used elsewhere too? We also use `recurse` on GET, could be nice to me consistent on this term.

Reply:
Totally agree with you :) The `recursive` parameter is available on the `/users` endpoint too.
I think `recursive` should be used instead of `recurse` (`recursive` means ... "recursive", this is not the case for `recurse`)

Reply:
I checked the OpenAPI documentation  (like here https://wazo-platform.org/documentation/api/configuration.html#tag/lines/operation/list_lines): `recurse` is used to indicate if the query should include sub-tenants or not.
Here (and in the new `/users` endpoint too) `recursive` is used to indicate that the update/delete/creation should update/delete/create the related resources/associations.
So I think we coud let `recursive` for this PR. What do you think @chartrandf ?

Reply:
All good, let's keep it like this